apps/docs/components/Mdx.tsx

```diff
@@ -17,7 +17,7 @@ export function Mdx({ content }: MdxProps) {
 				mdxOptions: {
 					// remarkPlugins: [remarkGfm, {}],
 					rehypePlugins: [
-						// [rehypeHighlight as any, {}],
+						[rehypeHighlight as any, {}],
```

Comment:
this will conflict with Sandpack - or are we getting rid of it? 
the nice thing about the sandpack stuff was that it was a stepping stone to having the snippets having some rendered preview stuff next to it in the docs, which would have happened down the line.

Reply:
Examples are still using it. They look ok to me! It's just the ``` stuff that's using pre/highlightjs.